exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/MetricAdapter.java

```diff
@@ -328,5 +331,24 @@ private static ByteString convertSpanId(String id) {
         OtelEncodingUtils.bytesFromBase16(id, SpanId.getLength()));
   }
 
+  // Fill labels too until Collector supports attributes and users have had a chance to update.
```

Comment:
NIT: mark as TODO so that it stands-out as something to change in the near future

Reply:
Not sure if it's exactly a TODO since it's not actionable until the collector updates. Any ideas on how best to track it and not forget?